pkg/envoy/proxy.go

```diff
@@ -116,7 +116,6 @@ func (e *envoy) args(fname string, epoch int, bootstrapConfig string) []string {
 		"--parent-shutdown-time-s", fmt.Sprint(int(convertDuration(e.Config.ParentShutdownDuration) / time.Second)),
 		"--service-cluster", e.Config.ServiceCluster,
 		"--service-node", e.Node,
-		"--max-obj-name-len", fmt.Sprint(e.Config.StatNameLength),
```

Comment:
we dont need this anymore?

Reply:
I added this long time ago to deal with cluster names longer than 63 characters

Reply:
Envoy removed it: https://github.com/istio/istio/issues/25595.